packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect, HTMLAttributes, ReactNode } from "react"
+import { v4 } from "uuid"
+import classnames from "classnames"
+import { history } from "prosemirror-history"
+import { keymap } from "prosemirror-keymap"
+import { EditorState } from "prosemirror-state"
+import { Node } from "prosemirror-model"
+import { Label } from "@kaizen/draft-form"
+import { baseKeymap } from "prosemirror-commands"
+import { buildKeymap, useRichTextEditor } from "@cultureamp/rich-text-toolkit"
+import { OverrideClassName } from "@kaizen/component-base"
+import { EditorContentArray } from "./types"
+import { createSchemaFromControls } from "./schema"
+import styles from "./RichTextEditor.scss"
+
+export interface RichTextEditorProps
+  extends OverrideClassName<Omit<HTMLAttributes<HTMLDivElement>, "onChange">> {
+  onChange: (content: EditorContentArray) => void
+  value: EditorContentArray
+  labelText: ReactNode
```

Comment:
I haven't added the ability to do `aria-labelledby` here yet instead because I'd quite like to do a discriminated union if I can, which I'll save for a separate piece of work.

Reply:
Card created for that here: https://cultureamp.atlassian.net/browse/KDS-356